libs/datamodel/core/tests/render_to_dmmf/parser_renderer_ast.rs

```diff
@@ -113,6 +113,24 @@ model Author {
     assert_rendered(input, input);
 }
 
+#[test]
+fn test_parser_renderer_native_types_via_ast() {
```

Comment:
You should add the same test case to `parser_render_dml`. I have the hunch that there's a small bug in there.